playpen/all_27.sh

```diff
@@ -0,0 +1,17 @@
+for repo in pulp pulp_puppet pulp_rpm
```

Comment:
How about adding a shebang to the top of this script?

What if I also have `pulp_python`, `pulp_docker`, etc cloned? Even if nothing needs to change, it'd probably be good to add a comment and explain that.